reconciler/reconciler.go

```diff
@@ -140,27 +138,25 @@ outer:
 		case <-refreshTimer.C:
 		}
 
-		now := time.Now()
+		durationUntilRefresh := r.config.RefreshInterval
 
 		// Iterate over the objects in revision order, e.g. oldest modification first.
 		// We look for objects that are older than [RefreshInterval] and mark them for
-		// pending in order for them to be reconciled again.
+		// refresh in order for them to be reconciled again.
 		iter := r.config.Table.LowerBound(r.DB.ReadTxn(), statedb.ByRevision[Obj](lastRevision+1))
```

Comment:
Are revisions starting from one? Otherwise it seems to me that the first object is skipped.

Reply:
Yes, revisions start from 1.